modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -96,6 +98,59 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+Starting from v1alpha3 (Operator version 0.4.0), you can mount directories from pre-created PVCs using the `spec.application.extraFiles.pvcs` field. PVCs are mounted as directories to the container's path, based on the following logic:
+
+* If `spec.application.extraFiles.pvcs[].mountPath` is defined, the PVC is mounted to the specified path.
+* If `spec.application.extraFiles.pvcs[].mountPath` is not defined, the PVC is mounted under `spec.application.extraFiles.mountPath/`.
```

Comment:
if neither above defined, use RHDH container WorkingDir (if defined) <- this is a new change

Reply:
Done